installer/install.sh

```diff
@@ -156,15 +160,15 @@ check_machine_image
 
 if [ "$fail" = "yes" ] && [ "$force" = "no" ] ; then
     echo "ERROR:$update_label: Machine mismatch"
-    echo "Running machine     : ${onie_arch}-${onie_machine}-${onie_machine_rev}"
-    echo "Update Image machine: ${image_arch}-${image_machine}-${image_machine_rev}"
+    echo "Running machine     : ${onie_arch}-${onie_build_machine}-${onie_machine_rev}"
+    echo "Update Image machine: ${image_arch}-${image_build_machine}-${image_machine_rev}"
```

Comment:
Should we need to add 'r' before `${onie_machine_rev}` and `${image_machine_rev}` here?